src/cmd/builtin/user.c

```diff
@@ -312,49 +330,57 @@ int cmd_user (optparse_t *p, int ac, char *av[])
-static struct optparse_subcommand user_subcmds[] = {
-    { "list",
-      "",
-      "List users and their assigned roles",
-      internal_user_list,
-      0,
-      NULL,
-    },
-    { "lookup",
-      "USERID",
-      "Lookup roles assigned to USERID",
-      internal_user_lookup,
-      0,
-      NULL,
-    },
-    { "addrole",
-      "USERID role[,role,...]",
-      "Add roles to USERID",
-      internal_user_addrole,
-      0,
-      NULL,
-    },
-    { "delrole",
-      "USERID role[,role,...]",
-      "Remove roles from USERID",
-      internal_user_delrole,
-      0,
-      NULL,
-    },
-    OPTPARSE_SUBCMD_END
-};
+static struct optparse_subcommand user_subcmds[] = {{
+                                                        "list",
+                                                        "",
+                                                        "List users and their assigned "
+                                                        "roles",
+                                                        internal_user_list,
+                                                        0,
+                                                        NULL,
+                                                    },
+                                                    {
+                                                        "lookup",
+                                                        "USERID",
+                                                        "Lookup roles assigned to "
+                                                        "USERID",
+                                                        internal_user_lookup,
+                                                        0,
+                                                        NULL,
+                                                    },
+                                                    {
+                                                        "addrole",
+                                                        "USERID role[,role,...]",
+                                                        "Add roles to USERID",
+                                                        internal_user_addrole,
+                                                        0,
+                                                        NULL,
+                                                    },
+                                                    {
+                                                        "delrole",
+                                                        "USERID role[,role,...]",
+                                                        "Remove roles from USERID",
+                                                        internal_user_delrole,
+                                                        0,
+                                                        NULL,
+                                                    },
+                                                    OPTPARSE_SUBCMD_END};
```

Comment:
This shows up several times and wondering if there's a way to deal with it.  Obviously, it's indenting the "proper way", but in these areas it's specifically indented for readability given it's a giant list of struct entries.

Reply:
My personal favorite is to add a comment to the last line that makes it almost 80 chars long.  Doing that and formatting it gives us this:

```c
static struct optparse_subcommand user_subcmds[] = {
    {
        "list",
        "",
        "List users and their assigned roles",
        internal_user_list,
        0,
        NULL,
    },
    {
        "lookup",
        "USERID",
        "Lookup roles assigned to USERID",
        internal_user_lookup,
        0,
        NULL,
    },
    {
        "addrole",
        "USERID role[,role,...]",
        "Add roles to USERID",
        internal_user_addrole,
        0,
        NULL,
    },
    {
        "delrole",
        "USERID role[,role,...]",
        "Remove roles from USERID",
        internal_user_delrole,
        0,
        NULL,
    },
    OPTPARSE_SUBCMD_END  // long comment here to make sure things stay put....
};
```

Equally we could mark them, but this is equally effective and actually fixes things in the middle if they get broken.

Reply:
Interestingly enough, this has its own option in clang-format as of version 13.